client/src/GamemasterDesktop/DesktopView.tsx

```diff
@@ -125,6 +107,9 @@ function Grimoire({ isPlayerView = true }: GrimoireProps) {
                   <SetCountModal
                     title="Votes to Execute:"
                     onSet={(votes: number) => setVotesToExecute(player, votes)}
+                    defaultValue={
+                      game.onTheBlock[player] ? game.onTheBlock[player] : 0
```

Comment:
You should be able to simplify this as `game.onTheBlock[player] ?? 0` I believe.